ts/session/messages/outgoing/content/TypingMessage.ts

```diff
@@ -1,7 +1,27 @@
 import { ContentMessage } from './ContentMessage';
 import { SignalService } from '../../../../protobuf';
+import { TextEncoder } from 'util';
+
+interface TypingMessageParams {
+  timestamp: number;
+  identifier: string;
+  isTyping: boolean;
+  typingTimestamp: number | null;
```

Comment:
try and prefer `undefined` over `null` where possible.

The distinction between the 2 is that `null` can be thought of as a concrete value (in JSON we need null) while `undefined` is not a concrete value

Reply:
Fixed